src/shadertools/modules/picking/README.md

```diff
@@ -32,22 +32,23 @@ main() {
     ... apply any filters on gl_FragColor ...
   gl_FragColor = picking_filterPickingColor(gl_FragColor);
 }
+
 ```
 
 ## JavaScript Functions
 
 ### getUniforms
 
-`getUniforms` returns an object with key/value pairs representing the uniforms that the `picking` module shaders need.
+`getUniforms` takes an object takes a set of key/value pairs, returns an object with key/value pairs representing the uniforms that the `picking` module shaders need.
 
-`getUniforms({enabled, })`
+`getUniforms(opts)`
+opts can contain following keys:
+* `pickingValid` (*boolean*) - When true current instance picking color is ignored, hence no instance is highlighted.
```

Comment:
API Audit mode on:
* Most modules have an `enabled` flag, allowing them to be turned off.
* Is this semantically different? If not, should we call it `pickingEnabled`?
* Why is picking ignored when valid is `true`?

Reply:
`pickingValid` is renamed to `pickingSelectedColorValid`, I just updated this doc too. Also it implied `pickingColor` is ignored when this flag (highlighting) is off.